src/CurrentViewNumServlet.java

```diff
@@ -0,0 +1,73 @@
+
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.ArrayList;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+/**
+ * Servlet implementation class CurrentViewNumServlet
+ */
+@WebServlet("/CurrentViewNumServlet")
+public class CurrentViewNumServlet extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+       
+    /**
+     * @see HttpServlet#HttpServlet()
+     */
+    public CurrentViewNumServlet() {
+        super();
+        // TODO Auto-generated constructor stub
+    }
+
+	/**
+	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
+	 */
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		ServletContext context = getServletContext();
+		String sessionID = null;
+		Cookie[] cookies = request.getCookies();
+		if (cookies != null) {
```

Comment:
maybe we can use this
`request.getSession().getId()`

maybe we can try 
`System.out.println(request.getSession().getId())`